keyboards/clueboard/66_hotswap/config.h

```diff
@@ -16,7 +16,6 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
 #pragma once
-#include "config_common.h"
```

Comment:
This is still needed. 

```suggestion
#include "config_common.h"
```